gui/src/components/Taipy/Slider.tsx

```diff
@@ -245,6 +282,8 @@ const Slider = (props: SliderProps) => {
                     marks={marks}
                     valueLabelFormat={getLabel}
                     orientation={horizontalOrientation ? undefined : "vertical"}
+                    getAriaLabel={Array.isArray(parsedDefaultValue) ? ()=>"" : undefined}
+                    aria-label={undefined}
```

Comment:
aria-label: does it work when not a range ?

Reply:
Yes but. It is back for non-range sliders.